bitnami/rabbitmq-cluster-operator/templates/NOTES.txt

```diff
@@ -0,0 +1,8 @@
+** Please be patient while the chart is being deployed **
+
+Watch the RabbitMQ Cluster Operator Deployment status using the command:
+
+    kubectl get deploy -w --namespace {{ .Release.Namespace }} -l app.kubernetes.io/name={{ include "common.names.name" . }},app.kubernetes.io/instance={{ .Release.Name }}
+
+{{ include "common.warnings.rollingTag" .Values.image }}
+{{ include "common.warnings.rollingTag" .Values.rabbitmqImage }}
```

Comment:
Please complete these NOTES with precise instructions to connect to RabbitMQ from inside/outside the cluster, watch the status, etc.

Reply:
Plus, we want to have a simliar experience as the upstream RabbitMQ Operator, which does not deploy any rabbitmqCluster object

Reply:
Fair enough